app-editors/nano/nano-4.9.3.recipe

```diff
@@ -39,7 +41,6 @@ BUILD_REQUIRES="
 	devel:libz
 	"
 BUILD_PREREQUIRES="
-	cmd:autoreconf
```

Comment:
Haven't checked this, but if this works without autoreconf I guess you could leave out libtoolize also
msgfmt and msgmerge are provided by the libintl package so I guess they shouldn't be needed here also

Reply:
Fails to build without those three.